src/client/Microsoft.Identity.Client/IConfidentialClientApplication.cs

```diff
@@ -96,8 +96,6 @@ AcquireTokenByAuthorizationCodeParameterBuilder AcquireTokenByAuthorizationCode(
         /// <summary>
         /// Use <see cref="IClientApplicationBase.GetAccountAsync(string)"/> in web apps and web APIs, and use a token cache serializer for better security and performance. See https://aka.ms/msal-net-cca-token-cache-serialization.
```

Comment:
Pls add a remark that this is only for use with ROPC

Reply:
we have no way to enforce that it's only used for ROPC in CCA, though
Are we sure that the cache is always partitioned?

Reply:
AFAIK the cache is always partitioned.

Reply:
It is partitioned. But please add a unit test (or update the integration test) @neha-bhargava, where you show that TokenCacheNotificationArgs has the correct `SuggestedCacheKey`.

Reply:
@neha-bhargava and I had a chat and concluded that `AcquireTokenForUsernamePassword` is partitioned, but `GetAccounts()` is not partitioned (and can't really be partitioned).

From MSAL perspective, we can ask ppl to use `GetAccount(id)`. 
For Id.Web, this gets a bit tricky.

Reply:
How would that work in MSAL? where do they get the id from? (assuming this is tid.oid?)?

Reply:
Proposing that CCA.AcquireTokenByUserPassword first checks the cache, and if not found or error, goes to ESTS.
Like OBO does.